src/cascadia/TerminalSettingsEditor/SettingContainer.cpp

```diff
@@ -82,6 +104,11 @@ namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
         {
             if (auto tb{ child.try_as<Controls::TextBlock>() })
             {
+                // TODO GH#6800: 'OverrideTarget' will be replaced with hyperlink/text directing the user to another profile.
+                // Create the override message
+                const auto overrideMsg{ fmt::format(std::wstring{ RS_(L"SettingContainer_OverrideIntro") }, RS_(L"SettingContainer_OverrideTarget")) };

```

Comment:
```suggestion
                const auto overrideMsg{ fmt::format(std::wstring_view{ RS_(L"SettingContainer_OverrideIntro") }, RS_(L"SettingContainer_OverrideTarget")) };
```

no need to copy the string just as a format input